fbw-a32nx/src/systems/systems-host/systems/FWC/PseudoFWC.ts

```diff
@@ -3150,18 +3163,21 @@ export class PseudoFWC {
     this.recallFailures.push(...recallFailureKeys);
     this.nonCancellableWarningCount = 0;
 
+    const simTime = this.simTime.get();
+
     // Failures first
     for (const [key, value] of Object.entries(this.ewdMessageFailures)) {
-      if (value.flightPhaseInhib.some((e) => e === flightPhase)) {
-        continue;
-      }
-
       // new warning?
       const newWarning =
         (value.side === 'LEFT' && !this.failuresLeft.includes(key) && !recallFailureKeys.includes(key)) ||
         (value.side === 'RIGHT' && !this.failuresRight.includes(key));
 
-      if (value.simVarIsActive.get()) {
+      if (newWarning && value.flightPhaseInhib.some((e) => e === flightPhase)) {
+        continue;
+      }
+
+      // consider monitor input confirm time (0.3 sec)
+      if (value.simVarIsActive.get() && simTime >= (this.ewdFailureActivationTime.get(key) ?? 0) + 0.3) {
```

Comment:
The 0.3s should be adjustable per warning, since we already have a few warnings that should have it set to 0

Reply:
That's true. I'll add an optional prop for it. If you have the codes that are 0 I can also do them in this PR, but I don't intend to go check every one.